ACE/ace/OS_NS_Thread.inl

```diff
@@ -2496,6 +2553,13 @@ ACE_OS::thr_getprio (ACE_hthread_t ht_id, int &priority, int &policy)
   return 0;
 # elif defined (ACE_HAS_VXTHREADS)
   return ::taskPriorityGet (ht_id, &priority);
+# elif defined (ACE_INTEGRITY)
+  Value active_prio;
+  const Error err = ::GetActivePriority (ht_id, &active_prio);
+  if (err != Success)
+    return -1;
```

Comment:
ACE_FAIL_RETURN?

Reply:
From its definition and places where it's used, looks like ACE_FAIL_RETURN is intended for Windows.